ambry-quota/src/main/java/com/github/ambry/quota/storage/StorageQuotaEnforcer.java

```diff
@@ -151,7 +151,10 @@ private QuotaRecommendation recommendBasedOnQuotaAndUsage(Pair<Long, Long> pair)
       // There is no quota set for the given account/container
       return NO_QUOTA_VALUE_RECOMMENDATION;
     }
-    QuotaAction quotaAction = (currentUsage >= quotaValue) ? QuotaAction.REJECT : QuotaAction.ALLOW;
+    QuotaAction quotaAction = QuotaAction.ALLOW;
+    if(config.shouldThrottle) {
+      quotaAction = (currentUsage >= quotaValue) ? QuotaAction.REJECT : QuotaAction.ALLOW;
```

Comment:
Nit: quotaAction has already default value of of QuotaAction.ALLOW. No need to overwrite it with the same value

Reply:
done